Objects/genericaliasobject.c

```diff
@@ -683,7 +683,7 @@ ga_iter_clear(PyObject *self) {
     return 0;
 }
 
-static PyTypeObject Py_GenericAliasIterType = {
+PyTypeObject _Py_GenericAliasIterType = {
```

Comment:
Would you mind to document that this type is only exported to be able to clear it in _PyTypes_FiniTypes()?